app/assets/javascripts/actions/projectBoard.js

```diff
@@ -57,6 +61,9 @@ export const sendErrorNotificationIfNeeded = (dispatch, code, condition) => {
   if (condition) dispatch(sendErrorNotification(code, { custom: true }));
 }
 
+export const reverseColumnsProjectBoard = () =>
+  dispatch =>  dispatch(reverseColumns())
```

Comment:
Why are you using `dispatch` here?
Because you could use `reverseColumns` instead of `reverseColumnsProjectBoard` and would work fine.